Lib/test/test_csv.py

```diff
@@ -1415,10 +1415,6 @@ def test_ordered_dict_reader(self):
 
 
 class MiscTestCase(unittest.TestCase):
-    def test__all__(self):
-        extra = {'__doc__', '__version__'}
-        support.check__all__(self, csv, ('csv', '_csv'), extra=extra)
-
```

Comment:
I think `test__all__` should be kept?


```suggestion
    def test__all__(self):
        support.check__all__(self, csv, ('csv', '_csv'))
```

Reply:
@AA-Turner Isn't `test__all__` a standard thing? Why is this override needed if we no longer have an `extra`?

Reply:
``test.support.check__all__()`` is, but ``test__all__`` isn't defined in ``unittest.TestCase``, so we do need to define the test to keep it.

Reply:
No, it is not overridden, it is added. It compares the `__all__` content with the calculated expected value (all public function, classes, etc defined in the specified modules). `extra` is an extra.